azure_functions_worker/dispatcher.py

```diff
@@ -290,6 +290,8 @@ async def _handle__worker_init_request(self, request):
 
         if DependencyManager.is_in_linux_consumption() \
                 and is_envvar_true(PYTHON_LOAD_FUNCTIONS_INIT):
+            logger.info(
+                "PYTHON_LOAD_FUNCTIONS_INIT enabled. Importing azure functions")
```

Comment:
On a second thought - shouldn't this info be within DependencyManager? Where it should make a call on loading or not.

Reply:
DependencyManager does nothing in this operation other than checking if the app is consumption or not. This log should be in the init request since its being imported at init. Also this is just a temp log till we validate that the functionality works and it will be removed once we make it default.